tests/test_models.py

```diff
@@ -140,3 +143,116 @@ def test_number_model_copy():
     c = HyComplex(42j)
     assert (c == copy.copy(c))
     assert (c == copy.deepcopy(c))
+
+
+PRETTY_STRINGS = {
+    k % ('[1.0] {1.0} (1.0) #{1.0} (0.0 1.0 . 2.0)',):
+        v.format("""
+  HyList([
+    HyFloat(1.0)]),
+  HyDict([
+    HyFloat(1.0)  # odd
```

Comment:
What do these comments with the text "odd" mean?

Reply:
It means there's an odd number of elements in your `HyDict`. This is allowed, since a `HyDict` is really a list, but it's a mistake if your macro expands to this, because a `dict` is built from pairs. So I wanted to make it very visible for debugging purposes.

Note also that `# odd` only shows up when PRETTY is on.

You might legitimately have an odd number in an intermediate stage of the macroexpansion though. So I don't want to add error checking to the `HyDict` methods to prevent that state.

Reply:
Oh, interesting. That makes sense.